doc/faq/index.rst

```diff
@@ -55,27 +55,29 @@ libraries are required.
 What version of Python can I use?
 ---------------------------------
 
++-----------------+------------------+---------------------------+
+| pydicom version |  Release date    | Python versions           |
++=================+==================+===========================+
+| 1.0             | March 2018       | 2.7, 3.4, 3.5, 3.6        |
++-----------------+------------------+---------------------------+
+| 1.1             | June 2018        | 2.7, 3.4, 3.5, 3.6        |
++-----------------+------------------+---------------------------+
+| 1.2             | October 2018     | 2.7, 3.4, 3.5, 3.6        |
++-----------------+------------------+---------------------------+
+| 1.3             | July 2019        | 2.7, 3.4, 3.5, 3.6        |
++-----------------+------------------+---------------------------+
+| 1.4             | January 2020     | 2.7, 3.5, 3.6, 3.7, 3.8   |
++-----------------+------------------+---------------------------+
+| 2.0             | May 2020         | 3.5, 3.6, 3.7, 3.8        |
++-----------------+------------------+---------------------------+
+| 2.1             | November 2020    | 3.6, 3.7, 3.8, 3.9        |
++-----------------+------------------+---------------------------+
+| 2.2             | August 2021      | 3.6, 3.7, 3.8, 3.9        |
 +-----------------+------------------+--------------------------+
-| pydicom version |  Release date    | Python versions          |
-+=================+==================+==========================+
-| 1.0             | March 2018       | 2.7, 3.4, 3.5, 3.6       |
-+-----------------+------------------+--------------------------+
-| 1.1             | June 2018        | 2.7, 3.4, 3.5, 3.6       |
-+-----------------+------------------+--------------------------+
-| 1.2             | October 2018     | 2.7, 3.4, 3.5, 3.6       |
-+-----------------+------------------+--------------------------+
-| 1.3             | July 2019        | 2.7, 3.4, 3.5, 3.6       |
-+-----------------+------------------+--------------------------+
-| 1.4             | January 2020     | 2.7, 3.5, 3.6, 3.7, 3.8  |
-+-----------------+------------------+--------------------------+
-| 2.0             | May 2020         | 3.5, 3.6, 3.7, 3.8       |
-+-----------------+------------------+--------------------------+
-| 2.1             | November 2020    | 3.6, 3.7, 3.8, 3.9       |
-+-----------------+------------------+--------------------------+
-| 2.2             | August 2021      | 3.6, 3.7, 3.8, 3.9       |
-+-----------------+------------------+--------------------------+
-| 2.3             | ~February 2022   | 3.6, 3.7, 3.8, 3.9, 3.10 |
-+-----------------+------------------+--------------------------+
+| 2.3             | March 2022       | 3.6, 3.7, 3.8, 3.9, 3.10  |
++-----------------+------------------+---------------------------+
+| 2.4             | ~September 2022  | 3.7, 3.8, 3.9, 3.10, 3.11 |
++-----------------+------------------+---------------------------+
```

Comment:
I removed 3.6 here as it is EOL since last year, but I'm not sure if this is correct - did we have some plan here?

Reply:
I don't recall if we had a plan, but it makes sense to drop it.